setup.py

```diff
@@ -42,6 +41,7 @@
     'jupyter>=1.0.0,<2',
     'rundoc>=0.4.3,<0.5',
     'pytest-subtests>=0.5,<1.0',
+    'psutil>=5.7,<6',
```

Comment:
Why do we need this? If we are removing the only place in the code where `psutil` is used, shouldn't we also be able to remove it from every test where it is mocked?

Reply:
Yes we can revise those test cases